code/modules/mob/living/carbon/xenomorph/xenoprocs.dm

```diff
@@ -247,11 +247,13 @@
 		return
 	if(evolution_stored >= xeno_caste.evolution_threshold || !(xeno_caste.caste_flags & CASTE_EVOLUTION_ALLOWED))
 		return
-	if(hive?.living_xeno_ruler)
```

Comment:
same here then have the hive proc check like a gamemode flag like XENOS_REQUIRE_RULER